languages/pl_pl/Settings/Vtiger.php

```diff
@@ -352,19 +352,14 @@
 	//Pluralized
 	'PLU_USERS_0' => 'Użytkownik',
 	'PLU_USERS_1' => 'Użytkowników',
-	'PLU_USERS_2' => 'Użytkowników',
```

Comment:
You can not delete this, in Polish it is needed
https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/vendor/yetiforce/Language.php#L256